### DISCUSSION TITLE
Stackable Sheets

### DISCUSSION THREAD

**Discussion by appinteractive** [fujidaiti/smooth_sheets#119] [Ideas]
I just stumbled upon that layout of stacked sheets and was wondering how something like that would be possible with this library. 

![grafik](https://github.com/fujidaiti/smooth_sheets/assets/432828/7a304cb3-4f38-4773-ac31-adab0e5f0aad)

https://github.com/fujidaiti/smooth_sheets/assets/432828/32b84328-2610-4bcb-9ee2-cc9fc8f71c3f

**Comment by fujidaiti** [comment]
It's like iOS-style modal sheets, but not full screen. What happens when we close the topmost sheet?

**Reply by appinteractive** [reply]
It returns to the previous one. I think the difference is also, that the background does not scale but only the previous sheet. 
Regarding the iOS like sheet, I still don't know If I can use it in my material app though.

**Reply by appinteractive** [reply]
@fujidaiti I updated the description with a recording of the behavior. I find it very compelling for some cases.

**Reply by fujidaiti** [reply]
Thanks. I'll convert this discussion to a future request.

**Comment by fujidaiti** [comment]
Closing this as moved to #122.
